tasks/jenkins.rake

```diff
@@ -3,17 +3,24 @@ require 'English'
 begin
   require "ci/reporter/rake/test_unit"
   namespace :jenkins do
+    desc 'Sets up pre-CI environment for testing'
     task :unit => ["jenkins:setup:test_unit", 'rake:test']
 
     namespace :setup do
+      desc 'Sets up pre-CI environment for Minitest'
       task :pre_ci do
         ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
         gem 'ci_reporter'
       end
-      task :minitest  => [:pre_ci, "ci:setup:minitest"]
+
+      desc 'Sets up pre-CI environment for MiniTest'
+      task :minitest => [:pre_ci, "ci:setup:minitest"]
```

Comment:
This repository doesn't use minitest so we can drop this. It's a left over from when we copied it over to multiple repositories.